### DISCUSSION TITLE
Meal Planner

### DISCUSSION THREAD

**Discussion by PascPeli** [nextcloud/cookbook#1215] [FAQ and user questions] [answered]
Hello! 

I wanted to ask if there is any internal discussion or future feature that will allow our beloved cookbook to also help us with meal planning. 
You know having recipes is one thing but most of the time we are missing the ingredients :) .

Thanks!!

**Comment by seyfeb** [comment, marked as answer]
Afaik, there is no "internal" discussion about this feature going on. There has been some open discussion in #127 though. Nearly all discussion on issues and features is done openly although via different channels (github & Matrix).

**Reply by PascPeli** [reply]
Thank you! 
Of course by internal I don't mean "hidden" but rather "advanced"
